src/functions/guild/getAutomodRuleId.js

```diff
@@ -0,0 +1,15 @@
+module.exports = async(d) => {
+    const data = d.util.aoiFunc(d);
+    const [guildID = d.guild?.id, automodRuleName, force = true] = data.inside.splits;
```

Comment:
Either we keep it as `name` or as `automodRuleName` I believe name is better since shorter.
This should be for all automod related functions (name) required property.

Reply:
This helps prevent less confusion and consistent

Reply:
fixed